ansible_ai_connect/users/views.py

```diff
@@ -170,9 +171,14 @@ def get_context_data(self, **kwargs):
                 else:
                     has_expired_plan = up
 
+        days_left: int = int(
+            ((has_active_plan.expired_at - timezone.now()).days) if has_active_plan else 0
```

Comment:
This an still be negative:
```
>>> from datetime import date
>>> expired_at = date.fromisoformat('2024-01-01')
>>> now = date.fromisoformat('2024-12-31')
>>> int((expired_at - now).days)
-365
```

Reply:
Yeah, I didn't really pay attention since in this case we don't display the information. But let's fix that.

Reply:
Sorry, my bad.. I just looked at [`UserPlan.is_active`](https://github.com/ansible/ansible-ai-connect-service/blob/dda8106987cffba1cd809d703400c2609f0ed111/ansible_ai_connect/users/models.py#L179). I really should get some work to do rather than bother you. Sorry.